frontend/packages/shipwright-plugin/src/components/build-list/BuildTable.tsx

```diff
@@ -52,16 +60,51 @@ export const BuildHeader = () => {
       props: { className: columnClassNames[3] },
     },
     {
-      title: '',
+      title: t('shipwright-plugin~LastRun'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
       props: { className: columnClassNames[4] },
     },
+    {
+      title: t('shipwright-plugin~LastRunStatus'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
+      props: { className: columnClassNames[5] },
+    },
+    {
+      title: t('shipwright-plugin~LastRunTime'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
+      props: { className: columnClassNames[6] },
+    },
+    {
+      title: t('shipwright-plugin~LastRunDuration'),
+      // sortField: 'status.message',
+      // transforms: [sortable],
+      props: { className: columnClassNames[7] },
+    },
+    {
+      title: '',
+      props: { className: columnClassNames[8] },
+    },
   ];
 };
 
-export const BuildRow: React.FC<RowFunctionArgs<Build>> = ({ obj: build }) => {
+const getLatestBuildRun = (buildRuns, build) => {
+  const filteredRuns = buildRuns.filter((buildRun) =>
+    buildRun.metadata.name.includes(build.metadata.name),
```

Comment:
The BuildRuns are connected to a Build via the label `build.shipwright.io/name`. So it should be similar to:

```suggestion
    buildRun.metadata.labels?.[BUILDRUN_TO_BUILD_REFERENCE_LABEL] === build.metadata.name,
```